CONTRIBUTING.md

```diff
@@ -27,6 +47,67 @@ pull request reviewed, the reviewer(s) may ask you to complete additional
 work, tests, or other changes before your pull request can be ultimately
 accepted.
 
+# For maintainers
+
+## Setting up environment
```

Comment:
```suggestion
## Setting up your local development environment
```

Similar to the previous comment, other contributing files we have use the "Setting up your local development environment" as a header. It was also inspired by Flutter's CONTRIBUTING headers, see [this](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment) and [this](https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment).